hail/python/test/hail/methods/test_impex.py

```diff
@@ -1263,6 +1272,25 @@ def test_import_bed(self):
 
         self.assertEqual(t.interval.collect(), hl.eval(expected))
 
+    def test_import_bed_recoding(self):
+        bed_file = resource('example1.bed')
+        bed = hl.import_bed(bed_file,
+                            reference_genome='GRCh38',
+                            contig_recoding={str(i): f'chr{i}' for i in [*range(22), 'X', 'Y', 'MT']})
+        bed._force_count()
+        nbed = bed.count()
+        i = 0
+        with open(bed_file) as f:
+            for line in f:
+                if len(line.strip()) != 0:
+                    try:
```

Comment:
Is this just a complicated way to count the lines in the file? I like your tests above better.

Reply:
I copied this from a test you added a year ago above ;). I hardcoded the expected count and added the type check.